plugins/nanospell/plugin.js

```diff
@@ -159,6 +167,12 @@
 		init: function (editor) {
 			var self = this;
 
+			// a lock to prevent multiple spellchecks
+			this._isBusy = false;
```

Comment:
I'm trying to think for a better name to describe what exactly is busy. Maybe `_spellcheckRequestInProgress`?